src/test/java/api/loans/CheckInByBarcodeTests.java

```diff
@@ -377,70 +377,6 @@ public void canCheckInAnItemTwice()
       itemFromResponse.getString("barcode"), is("565578437802"));
   }
 
-  @Test
-  public void patronNoticeOnCheckInIsSentWhenCheckInLoanNoticeIsDefinedAndLoanExists()
```

Comment:
Can you help me understand the context behind removing these tests? I don't really understand this change so I'd like to understand the intent better.

Reply:
The point is that a library user should receive one email that includes all information and items from the library based on "end action session" button (or by time-out). In previous implementation user used to receive several emails or an email for each item.

Reply:
Ok, so these tests were for old behaviour and the new behaviour is tested elsewhere?

Reply:
For the new behavior I added test EndPatronActionSessionTests.java #checkInSessionShouldBeEndedWithPatronNotice()

Reply:
Given that notices should not be sent on check in any more, could there be tests for the absence of notices after a check in?

I’m aware that checking for the absence of something could be brittle. In this case it seems important that notices are only sent at the end of the session.

Reply:
Updated. 
I returned this test back and updated it for checking that notice was not sent